spec/BowlingSpec.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-undef */
+'use strict'
+describe('Bowling', function () {
+  var bowling
```

Comment:
I don't see any semicolons at the end of your declarations. In some places you may need them.